lib/ae_mdw/db/sync/transaction.ex

```diff
@@ -213,18 +212,14 @@ defmodule AeMdw.Db.Sync.Transaction do
          tx_hash,
          _block_index,
          block_hash,
-         mb_events
+         _mb_events
        ) do
     contract_pk = :aect_create_tx.contract_pubkey(tx)
     owner_pk = :aect_create_tx.owner_pubkey(tx)
-    events = Map.get(mb_events, tx_hash, [])
 
     :ets.insert(:ct_create_sync_cache, {contract_pk, txi})
 
-    mutations = [
-      ContractEventsMutation.new(contract_pk, events, txi)
```

Comment:
`:create_contract_tx` can emit events too (when contract `init` is called).

Reply:
Done